### ISSUE TITLE
Changing action bar style makes its elements lose the bright effect when pressed

### ISSUE THREAD

**Issue by barondem** [negusoft/holoaccent#34]
Hi! I'm making my first steps with HoloAccent and I found a problem: if I specify any action bar related style other than accentColor(normal, dark and action bar), like for example a 9 patch background for the action bar or a color background to solve the launch activity glitch as indicated in the wiki

**Comment by barondem** [comment]
(sorry the text wasn't complete) I lose the slight lightning effect on action bar elements (action views, overflow, home button) usually present. Am I missing something or is it a bug?
I use Android Studio and I'm testing on a Kit Kat Nexus 4.

**Comment by blurkidi** [comment]
I am afraid that I don't really understand the issue. You lose the "lighting effect", you mean the background color when you press an item? And it happens only when you assign a 9patch as the actionbar background?

In any case, my advice is to go step by step:
- Make the basic integration of HoloAccent (withouth the glitch fix).
- If it works, make the glitch fix with the color you you want.
- Then, if it is still fine, make the background your 9-path.

You can maybe share some code that represents the issue.

**Comment by barondem** [comment]
For light effect I mean this:
![img_20140801_140635](https://cloud.githubusercontent.com/assets/6568392/3782240/87480370-199f-11e4-91a0-a19418170920.jpg)
![img_20140801_140442](https://cloud.githubusercontent.com/assets/6568392/3782244/8c221868-199f-11e4-8fd0-8290540b0281.jpg)
![img_20140801_140410](https://cloud.githubusercontent.com/assets/6568392/3782250/919e1436-199f-11e4-8817-404257d74029.jpg)
I'm using ColoredActionBar Light and if I specify only accentColor\* attributes that effect it still present (I can also customize it with accentColorActionBar), but if I specify an element like android:actionBarStyle (with as value a style which defines as background a color or a 9 patch) or coloredActionBarBackgroundStyle (with as value a style which defines attributes like accentLineColor and accentBackgroundColor) I lose that effect. If I apply the action bar style to a common holo theme, I don't have any problem...

**Comment by barondem** [comment]
Solved, it was my fault and a misunderstanding with the colours: my nine patch was essentially of the same color that I had specified as "accentColor", but holo accent used as action bar background an automatically dark version of that color and as light effect a color very similar to my background color, so when I used my nine patch the light effect was still there but almost unnoticeable. Fixed anyway. This could be closed.
